postman/Public Collection.postman_environment.json

```diff
@@ -21,17 +21,23 @@
 			"enabled": true
 		},
 		{
-			"key": "AdminClientId",
+			"key": "ClientId",
 			"value": "<string>",
 			"type": "text",
 			"enabled": true
 		},
 		{
-			"key": "AdminClientSecret",
+			"key": "ClientSecret",
 			"value": "<string>",
 			"type": "text",
 			"enabled": true
 		},
+		{
+			"key": "audience",
+			"value": "<url string>",
+			"type": "default",
+			"enabled": true
+		},
```

Comment:
<img width="880" alt="Screen Shot 2022-09-22 at 8 17 48 AM" src="https://user-images.githubusercontent.com/98990619/191758562-f634362a-295c-4864-97bd-0f90e9a6cd18.png">

Reply:
nice, thanks!

Addition to this request - can we add the ENV variable values to be csdemo0 values?  prepopulate the baseurl, url, and audience values?  (i guess add audience as an Env Var too)

Reply:
will do.